pymc_marketing/clv/plotting.py

```diff
@@ -332,33 +332,26 @@ def plot_probability_alive_matrix(
         max_recency=max_recency,
     )
     # FIXME: This is a hotfix for ParetoNBDModel, as it has a different API from BetaGeoModel
-    #  We should harmonize them!
+    transaction_data = pd.DataFrame(
+        {
+            "customer_id": np.arange(mesh_recency.size),  # placeholder
+            "frequency": mesh_frequency.ravel(),
+            "recency": mesh_recency.ravel(),
+            "T": max_recency,
+        }
+    )
     if isinstance(model, ParetoNBDModel):
```

Comment:
Can we remove these if statements with this channel? If not, what is missing?

Also, can we do something similar in the function above https://github.com/ColtAllen/pymc-marketing/blob/bgnbd_api/pymc_marketing/clv/plotting.py#L227 ?

Reply:
I don't want to introduce too much scope creep in this PR, and this would be the very next thing I work on after the `BetaGeoModel` changes get merged.

Reply:
ok! makes sense!